templates/mongodb-atlas.base.template.yaml

```diff
@@ -193,31 +174,33 @@ Resources:
           Principal:
             AWS: !Sub "arn:aws:iam::${AWS::AccountId}:root"
           Action: 'sts:AssumeRole'
+        - Effect: Allow
+          Principal:
+            AWS: !Sub "arn:aws:iam::${AWS::AccountId}:root"
+            Service: ecs-tasks.amazonaws.com
+          Action: 'sts:AssumeRole'
+      ManagedPolicyArns:
+        - !Sub arn:${AWS::Partition}:iam::aws:policy/service-role/AmazonECSTaskExecutionRolePolicy
+        - !Sub arn:${AWS::Partition}:iam::aws:policy/AmazonEC2ContainerRegistryReadOnly
```

Comment:
Why do you need this change?

Reply:
@andreaangiolillo see the comment here:
https://github.com/aws-quickstart/quickstart-mongodb-atlas-mean-stack-aws-fargate-integration/pull/1/files#r1112151978